pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -46,8 +47,17 @@ var _ Interface = new(cadvisorClient)
 // The amount of time for which to keep stats in memory.
 const statsCacheDuration = 2 * time.Minute
 const maxHousekeepingInterval = 15 * time.Second
+const defaultHousekeepingInterval = 10 * time.Second
 const allowDynamicHousekeeping = true
 
+func init() {
+	// Override the default cAdvisor housekeeping interval.
+	if f := flag.Lookup("housekeeping_interval"); f != nil {
```

Comment:
I chatted with @timstclair offline. This PR will go in as-is. We will announce to the community that cAdvisor flags will be deprecated, and then proceed with refactoring of flags and hiding them from Kubelet.